metadatastore/commands.py

```diff
@@ -157,6 +157,7 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     if project is None:
         project = ''
 
+    beamline_config = BeamlineConfig.objects(uid=beamline_config).first()
```

Comment:
ok then.  `[0]`  it is, but it probably should also get a comment.

I guess the use-case for `get` is in cases where you have an optional field where the non-null entries should be unique (and you would like to know when that is not true).  Looking at the code, `get` does a bunch of extra work to raise in the case were there is more than one result.  We are ok with out this check because the field is enforced to be unique by mongo already.